src/Storages/MergeTree/MergeTreeIndexText.h

```diff
@@ -128,44 +139,39 @@ struct PostingsSerialization
         RawPostings = 1ULL << 0,
         /// If set, the posting list is embedded into the dictionary block to avoid additional random reads from disk.
         EmbeddedPostings = 1ULL << 1,
+        /// If set, the posting list is serialized as a single block.
```

Comment:
I think in l. 142, we should say "If unset, the number of blocks is stored as an additional VarUInt."